doc/reference/bluetooth/mesh/shell.rst

```diff
@@ -372,6 +372,15 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
 	* ``interval``: Sets the new relay retransmit interval in milliseconds if ``val`` is ``on``. Ignored if ``val`` is ``off``. Defaults to ``20`` if omitted.
 
 
+``mesh cmd_net_transmit [<count: 0-7> <interval: 10-320>]``
```

Comment:
Sorry, missed this earlier, cmd_net_transmit is the function, but the command is `net-transmit-param`
```suggestion
``mesh net-transmit-param [<count: 0-7> <interval: 10-320>]``
```